src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -37,6 +41,63 @@ public AWSElasticBeanstalkHandler(IAWSClientFactory awsClientFactory, IOrchestra
             _optionSettingHandler = optionSettingHandler;
         }
 
+        /// <summary>
+        /// Elastic Beanstalk Windows deployments require a manifest file to be included with the application to be deployed.
+        /// This function creates the windows manifest file and adds it to the zip file that will be deployed to Beanstalk.
+        /// </summary>
+        public void SetupWindowsDeploymentManifest(Recommendation recommendation, string dotnetZipFilePath)
```

Comment:
Can you add a unit test for this method?

Reply:
Done